### DISCUSSION TITLE
Timeline for Casper 2.0 mainnet deploy

### DISCUSSION THREAD

**Discussion by cdc-dw** [casper-network/condor-info#51] [Q&A] [answered]
What's the planned timeline for the deploy to mainnet?

**Comment by melpadden** [comment, marked as answer]
Short answer: It depends on project readiness. It won't be this month, check back around the 15th of each month. 

The way the Condor rollover to Mainnet is scheduled is as follows: On the 15th of each month the Casper Association, in collaboration with stakeholders and project leads, assesses the readiness of each relevant entity with respect to Condor. If all the relevant entities are at a sufficient state of readiness, the decision will be taken to deploy Condor to Mainnet. The deployment will then be scheduled for the end of that month. 

When the decision is made to rollover, announcements will be made on this forum as well as other channels.

**Reply by GuybrushX** [reply]
I think it's important to mention that Testnet will be first and only after it's tested there successfully, considered as safe and stable it will rollout to Mainnet. Or will that process change?

Usually Testnet was tested for ~1 week in the past - I can imagine this could or even should be longer for such a big upgrade like Condor.
